shallow_backup.py

```diff
@@ -156,7 +161,7 @@ def backup_dotfiles(backup_path):
 
 	# get dotfolders and dotfiles
 
-	dotfiles_for_backup = [".bashrc", ".bash_profile", ".gitconfig", ".pypirc", ".shallow-backup", ".zshrc"]
+	dotfiles_for_backup = [".bashrc", ".bash_profile", ".gitconfig", ".pypirc", ".shallow-backup", ".zshrc", ".vimrc", ]
```

Comment:
There's an extra `,` at the end of this list which may (?) cause an error. I'm not sure, but let's remove this to be safe.